Dockerfile

```diff
@@ -120,7 +120,7 @@ RUN git clone https://github.com/NVIDIA/TensorRT-LLM.git && \
 
 RUN cd TensorRT-LLM && \
     pip install ./build/tensorrt_llm*.whl
-ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/cuda-12/compat/lib.real/
```

Comment:
discussed offline, but @ko3n1g to check why this is now not working in CI

Reply:
So this image passed the unit test suite on CW: 
```
0.00s call     tests/test_trainer_utils.py::test_check_progress[input_kwargs5-expected_result5]
====================================================================================================== 55 passed in 1650.18s (0:27:30) =======================================================================================================
root@cw-dfw-h100-004-315-012:/opt/NeMo-Aligner#
```

It took much longer though than our VM's H100. Not sure what to make of it

Reply:
So this image passed the unit test suite internally: 
```
0.00s call     tests/test_trainer_utils.py::test_check_progress[input_kwargs5-expected_result5]
====================================================================================================== 55 passed in 1650.18s (0:27:30) =======================================================================================================
root@XXX:/opt/NeMo-Aligner#
```

It took much longer though than our VM's H100. Not sure what to make of it

Reply:
Created https://github.com/NVIDIA/NeMo-Aligner/issues/336 to track